build_image.py

```diff
@@ -186,25 +187,53 @@ def create_dockerfile_base(config: OfrakImageConfig) -> str:
 
     # Support multi-stage builds
     for package_path in config.packages_paths:
-        if not os.path.exists(os.path.join(package_path, "Dockerstage")):
+        dockerstage_path = os.path.join(package_path, "Dockerstage")
+        if not os.path.exists(dockerstage_path):
             continue
-        with open(os.path.join(package_path, "Dockerstage")) as file_handle:
+        with open(dockerstage_path) as file_handle:
             dockerstub = file_handle.read()
-        dockerfile_base_parts.append(dockerstub)
+        dockerfile_base_parts += [f"### {dockerstage_path}", dockerstub]
 
     dockerfile_base_parts += [
         "FROM python:3.7-bullseye@sha256:338ead05c1a0aa8bd8fcba8e4dbbe2afd0283b4732fd30cf9b3bfcfcbc4affab",
+        "",
     ]
+
+    requirement_suffixes = [""]
+    if config.install_target == InstallTarget.DEVELOP:
```

Comment:
```suggestion
    if config.install_target is InstallTarget.DEVELOP:
```
Enum members are compared by identity: https://docs.python.org/3/howto/enum.html#comparisons.

Reply:
Done